schemas/README.md

```diff
@@ -29,11 +37,29 @@ Generated by https://github.com/foxglove/schemas
 - [PosesInFrame](#posesinframe)
 - [Quaternion](#quaternion)
 - [RawImage](#rawimage)
+- [SceneEntity](#sceneentity)
+- [SceneEntityDeletion](#sceneentitydeletion)
+- [SceneEntityUpdate](#sceneentityupdate)
+- [SpherePrimitive](#sphereprimitive)
+- [TextPrimitive](#textprimitive)
+- [TriangleListPrimitive](#trianglelistprimitive)
 - [Vector2](#vector2)
 - [Vector3](#vector3)
 
 ----
 
+## enum LineType
+
+An enumeration indicating how input points should be interpreted to create lines
+
+name | value | description
+---- | ----- | -----------
+`LINE_STRIP` | 0 | 0-1, 1-2, ..., (n-1)-n
+`LINE_LOOP` | 1 | 0-1, 1-2, ..., (n-1)-n, n-0
+`LINE_LIST` | 2 | 0-1, 2-3, 4-5, ...
```

Comment:
`LINE_LIST` seems unnecessary, since you could just add multiple Lines to your SceneEntity.

That would mean you could remove this whole enum and just have a `loop` boolean on the Line schema.

Reply:
LINE_LIST is a rendering primitive. The difference between rendering a LINE_LIST from a set of points in one message vs rendering 1000 lines (imagine lane lines) from 1000 messages is both a massive message overhead and rendering overhead.

Its better to think about these messages as giving you ways to access rendering primitives.

Reply:
I see - GL refers to this as `LINES`.

https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/drawElements

Should we stick to the standard naming, or do we think our name is less confusing? For me personally our name seems less confusing, but I'm not a 3d developer.